script.js

```diff
@@ -65,10 +72,95 @@ function showTitle() {
+  tvMazeWrapper.classList.add("tv-maze-wrapper");
+  body.appendChild(tvMazeWrapper);
+  const tvMazeLink = document.createElement("a");
+  tvMazeLink.setAttribute("target", "_blank");
+  tvMazeLink.setAttribute("rel", "noreferrer noopener");
+  tvMazeLink.classList.add("tv-maze-link");
+  tvMazeLink.innerHTML = "SOURCED FROM TV-MAZE.COM";
+  tvMazeLink.href = "https://www.tvmaze.com";
+  tvMazeWrapper.appendChild(tvMazeLink);
+}
+
+/***** CREATE SEARCH SECTION *****/
+function searchAreaWrapper() {
+  searchBarWrapper.classList.add("search-bar-wrapper");
+  body.appendChild(searchBarWrapper);
+}
+
+/***** CREATE SEARCH BAR*****/
+function searchBar() {
+  searchInput.classList.add("search-input");
+  searchInput.type = "text";
+  searchInput.setAttribute("placeholder", "Search Shows");
+  searchInput.setAttribute("Name", "searchBar");
+  searchInput.setAttribute("label", "search-episodes");
+  searchBarWrapper.appendChild(searchInput);
+}
+
+/***** SHOWS DROPDOWN OPTIONS *****/
+function allShowsDropdown() {
+  // const showDropdownWrapper = document.createElement("form");
+  showDropdownWrapper.setAttribute("id", "show-dropdown-wrapper");
+  showDropdownWrapper.classList.add("dropdown-form");
+  showDropdownSelection.classList.add("dropdown-selector");
+  showDropdownSelection.setAttribute("name", "show-dropdown-selection");
+  showDropdownSelection.setAttribute("label", "select-show");
+  searchBarWrapper.appendChild(showDropdownWrapper);
+  showDropdownWrapper.appendChild(showDropdownSelection);
+
+  //POPULATE SHOW OPTIONS
+  const arrShowsDropdown = [...allShows];
+  sortedShows = arrShowsDropdown.sort((a, b) => (a.name < b.name ? -1 : 1));
+  sortedShows.unshift({ id: "default" });
+  console.log(arrShowsDropdown);
+
+  sortedShows.forEach((show, index) => {
+    const showOptions = document.createElement("option");
+    showOptions.setAttribute("label", `${show.name}`);
+
+    if (index === 0) {
+      showOptions.setAttribute("selected", "selected");
+      showOptions.setAttribute("label", `List all shows`);
+    }
+
+    showOptions.setAttribute("value", show.id);
+    showDropdownSelection.appendChild(showOptions);
+  });
+}
+
+/***** SHOW & EPISODE COUNTER *****/
+function showsAndEpisodeCounter() {
+  counterWrapper.classList.add("counter-wrapper");
+  counterWrapper.appendChild(countH2);
+  searchBarWrapper.appendChild(counterWrapper);
+  countH2.classList.add("counter-heading");
+}
+
+/***** SHOW COUNT TEXT *****/
+function showCount(arr, arr2) {
+  countH2.textContent = `${arr.length} / ${arr2.length} Shows`;
+}
+
+/***** EPISODE COUNT TEXT *****/
+function episodeCount(arr, arr2) {
+  countH2.textContent = `${arr.length} / ${arr2.length} Episodes`;
+}
+
+/***** ALL CARDS WRAPPER *****/
+function createCardsWrapper() {
+  body.appendChild(cardsWrapper);
+  cardsWrapper.classList.add("cards-wrapper");
+}
+
 /***** FORMAT SHOW DATA *****/
 function formatShowData(seasonNum, episodeNum) {
```

Comment:
I really like the fact that you created a separate function to format the display of episodes' numbers.